lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -79,13 +85,13 @@
   }
 
   .nx-page-header {
-    position: fixed;
+    position: sticky;
     top: 0;
     left: 0;
     z-index: 1;
   }
 
   .nx-page-content {
-    margin: $nx-main-header-height auto 0 auto;
+    margin: 0 auto;
```

Comment:
While working on this, I discovered that switching between page scrolling and section scrolling was moving the page contents up and down by 1px.  It turns out the margin here was not set to the correct amount as it failed to consider the bottom border on the header.  Setting the position on the `.nx-page-header` to sticky avoids the need for a specific number here.